packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/example.html

```diff
@@ -46,3 +46,9 @@ <h2>Example</h2>
     </gux-form-field-radio>
   </fieldset>
 </form>
+
+<style>
+  .example-fieldset gux-form-field-radio {
+    margin-bottom: var(--gse-core-spacing-4xs);
```

Comment:
so are consumers supposed to use the token themselves like this then whenever they use the component?

Reply:
@jordanstith15 No, there is a radio-group component in the works which will handle the spacing. This is just there for the example.